abstracts.json

```diff
@@ -17,6 +17,6 @@
+{"_id": "mouginot.Recipe", "coauthors": "P.P.H. Wilson", "email": "", "firstname": "Baptiste", "institution": "", "lastname": "Mouginot", "references": "", "text": "The U.S. Department of Energy (DOE) Fuel Cycle Options (FCO) campaign has chartered studies to assess the future of the U.S. nuclear fleet using different criteria, e.g., nuclear waste management, resource utilization, and environmental impact. In the Evaluation and Screening phase [1], only equilibrium scenarios were considered for fuel cycles that were regarded as representative of a set of similar fuel cycle concepts. Each set of concepts represents a so-called Evaluation Group (EG). This work is based on a transition from the actual US nuclear fleet (EG01) to a steady state scenario of EG30. EG30 is a closed double strata scenario with light water and fast reactors. On one stratum, sodium fast reactors (SFRs) recycle their own transuranium elements (TRU). The SFRs also recycle the TRU produced by light water reactors (LWRs), which comprise the other stratum. The SFRs employ a blanket filled with natural uranium. The TRU produced by the SFR blanket is diluted with natural uranium to build the MOX fuel used in the LWRs. During the transition, both SFRs and LWRs are able to use low enriched uranium as a startup fuel. The TRU from the blanket can also been used as a filler to build the SFR MOX if needed.\n\nThis study aims to compare the transition calculation of a fixed deployment schedule performed with a recipe-based approach against one that employs in-simulation fuel fabrication and depletion modelisation. Both calculations were performed with the Cyclus fuel cycle simulator [2]. The recipe-based approach required fixed material flows (the transition batches correspond to a mix between the TRU from both the SFR recycled fuel and the SFR blanket), as well as 5 consecutive reprocessing passes to reach equilibrium with regards to the SFR. In contrast the in-simulation modeling scenario features mixing of the different material streams according to their availability and the reactor requirements. This means that depletion calculations are performed for each loaded fuel set. These models have been specially computed based on the previous CLASS models [3,4]. The two different calculations will be compared, specially in term of transition speed, unused plutonium inventories, and loaded fuel composition.", "timestamp": "", "title": "Recipe vs. Model"}
```

Comment:
Please turn the `m` into an `M`.  Also, you'll need to change the other repo too.